amd_openvx/openvx/ago/ago_haf_cpu_canny.cpp

```diff
@@ -76,6 +76,7 @@ int HafCpu_CannySobel_U16_U8_3x3_L1NORM
 		{
 			vx_int16 Gx = (vx_int16)srow0[x + 1] - (vx_int16)srow0[x - 1] + (vx_int16)srow2[x + 1] - (vx_int16)srow2[x - 1] + 2 * ((vx_int16)srow1[x + 1] - (vx_int16)srow1[x - 1]);
 			vx_int16 Gy = (vx_int16)srow2[x - 1] + (vx_int16)srow2[x + 1] - (vx_int16)srow0[x - 1] - (vx_int16)srow0[x + 1] + 2 * ((vx_int16)srow2[x] - (vx_int16)srow0[x]);
+			Gy = ~Gy + 1;
```

Comment:
please comment why you need to add Gy = ~Gy+1

Reply:
Similar change exist for t1 in SSE implementation of this function.
Please see: 
https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/7bd32f3794c17c8fb5829900c0e14eaa8a7f5fd1/amd_openvx/openvx/ago/ago_haf_cpu_canny.cpp#L111